docs/java-rest/usage.asciidoc

```diff
@@ -1,11 +1,19 @@
 == Getting started
 
-=== Maven Repository
+=== Maven Central Repository
 
 The low-level Java REST client is hosted on
 http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22org.elasticsearch.client%22[Maven
 Central]. The minimum Java version required is `1.7`.
 
+The low-level REST client is subject to the same release cycle as
+elasticsearch. Replace the version with the desired client version, first
+released with `5.0.0-alpha4`. There  is no relation between the client version
+and the elasticsearch version that the client can communicate with. The
+low-level REST client is compatible with all elasticsearch versions.
+
+=== Maven coordinates
```

Comment:
Agreed. I did not notice that chunk size is 1 for this repo. So I'll put that text back on both pages.